app/models/user/reputation_tokens/concept_author_token.rb

```diff
@@ -0,0 +1,22 @@
+class User::ReputationTokens::ConceptAuthorToken < User::ReputationToken
+  params :authorship, :concept
+  category :authoring
+  reason :authored_concept
+  value 10
```

Comment:
I've made authoring a concept have half the reputation of authoring a concept exercise. Too high still?

Reply:
No, I think that's ok, especially as the people who do one will tend to do both.